Source/Session.swift

```diff
@@ -74,7 +74,7 @@ open class Session {
         delegate.stateProvider = self
     }
 
-    public convenience init(configuration: URLSessionConfiguration = .alamofireDefault,
+    public convenience init(configuration: URLSessionConfiguration = URLSessionConfiguration.af.default,
```

Comment:
`URLSessionConfiguration.af.default` -> `.af.default`?

Reply:
Unfortunately that doesn't seem to work. Using the `.af.default` results in a compiler error about needing more type context.